collection_modules/btle_beacon/config/module.conf

```diff
@@ -1,10 +1,17 @@
-[CollectionPointConfig] 
+[ModuleConfig] 
```

Comment:
that was changes in the last merge? how did it come back? in fact the whole config looks old and has none of my advanced logging flags?

Reply:
not only that but the config is also missing the ability to pull ranges of major and minors which was required by TON

Reply:
This is the merge of your work and mine. Neither our work has been merged to master yet since this PR has been open for several days.